content/en/tracing/advanced/runtime_metrics.md

```diff
@@ -108,9 +120,11 @@ Additional JMX metrics can be added using configuration files that are passed to
 {{% /tab %}}
 {{% tab "Python" %}}
 
-Coming Soon. Reach out to [the Datadog support team][1] to be part of the beta.
+{{< get-metrics-from-git "python" >}}
 
-[1]: /help
+Along with displaying these metrics in your APM Service Page, Datadog provides a [default Python Runtime Metrics Dashboard][1] with the `service` and `runtime-id` tags that are applied to these metrics.
+
+[1]: https://app.datadoghq.com/dash/integration/xyz/python-runtime-metrics
```

Comment:
TODO: We need to replace `xyz` with the actual dashboard id.

Reply:
Good catch. Python Dashboard's up at https://app.datadoghq.com/dash/integration/30267/python-runtime-metrics right?

Reply:
```suggestion
[1]: https://app.datadoghq.com/dash/integration/30267/python-runtime-metrics
```